bcdaworker/worker/utils/common.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"github.com/CMSgov/bcda-app/bcda/models"
+	"github.com/CMSgov/bcda-app/bcda/models/fhir/alr"
+)
+
+func AlrSlicer(alrModels []*models.Alr, c chan *alr.AlrFhirBulk, limit int, bbbasepath string) {
+	for ;len(alrModels) > limit; {
```

Comment:
semicolon?

Reply:
Yea those are not necessary. Will remove.